node/README.md

```diff
@@ -80,7 +80,7 @@ Subcommand: run
   -c, --certificate  <arg>      Path to node's X.509 certificate file, that is
                                 being used for identification
   -d, --data_dir  <arg>         Path to data directory. Defaults to
-                                /var/lib/rnode
+                                $HOME/.rnode
```

Comment:
If I am correct I see that the docker is run with `--data_dir=/var/lib/rnode` . This is different right?

Reply:
Ok, thanks :+1: